content/en/docs/contribute/localization.md

```diff
@@ -119,11 +136,9 @@ mkdir -p content/de/docs/tutorials
 cp content/en/docs/tutorials/kubernetes-basics.md content/de/docs/tutorials/kubernetes-basics.md
 ```
 
-For an example of a localization-related [pull request](../create-pull-request), [this pull request](https://github.com/kubernetes/website/pull/10471) to the [Kubernetes website repo](https://github.com/kubernetes/website) added Korean localization to the Kubernetes docs.
+### Source files
 
-### Source Files
-
-Localizations must use English files from the most recent release as their source. The most recent version is **{{< latest-version >}}**.
+Localizations must be based on the English files from the most recent release. The most recent version is **{{< latest-version >}}**.
 
```

Comment:
nit: Possible change to avoid repeating recent release/version,
Localizations must be based on the English files from the most recent release,  {{< latest-version >}}.
nit: the double asterisk markup is not resolving on lines 141 and 148. If you use percents, ** {{ % latest-version % }} **, will that fix the problem?